src/components/NavigationBar/index.tsx

```diff
@@ -59,6 +61,13 @@ function NavigationBar() {
           />
         </div>
         <div>
+          <div>
+            {productList.length ? (
+              <p className={styles.cartNumber}>{productList.length}</p>
+            ) : (
+              <></>
+            )}
+          </div>
```

Comment:
Instead of using an empty fragment, i think you can do the following

`{productList.length && <p className={styles.cartNumber}>{productList.length}</p>}`

And that's it